.gitignore

```diff
@@ -83,6 +83,7 @@ fabtests/multinode/fi_*
 fabtests/prov/efa/src/fi_efa*
 fabtests/prov/lpp/src/lpp_regression
 pingpong/fi_*
+libefagda.so
```

Comment:
This looks odd. Third-party dependencies should not appear in this ignore, and also it means that .so file has to be in `libfabric/`, which is not necessary, right?